maths/trapezoidal_rule.py

```diff
@@ -20,29 +20,70 @@ def method_1(boundary, steps):
     y += (h / 2.0) * f(a)
     for i in x_i:
         # print(i)
-        y += h * f(i)
+        y += h * f(func, i)
     y += (h / 2.0) * f(b)
     return y
 
 
 def make_points(a, b, h):
-    x = a + h
-    while x < (b - h):
+    """
+    Yields equally spaced points between a and b
+    >>> x_i = make_points(0, 0.4, 0.1)
+    >>> next(x_i)
+    0.1
+    >>> next(x_i)
+    0.2
+    >>> next(x_i)
+    0.3
+    >>> next(x_i)
+    Traceback (most recent call last):
+        ...
+    StopIteration
+
+    >>> x_i = make_points(0, 1, -0.2)
+    >>> next(x_i)
+    Traceback (most recent call last):
+        ...
+    ValueError: h must be positive
+
+    >>> x_i = make_points(1,0,0.2)
```

Comment:
```suggestion
    >>> x_i = make_points(1, 0, 0.2)
```
Minor formatting stuff